slides/incentives_and_consensus_attacks.tex

```diff
@@ -0,0 +1,360 @@
+% Choose one to switch between slides and handout
+%\documentclass[]{beamer}
+\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Incentives and Potential Consensus Attacks}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Introduction}
+
+By now, we know about the central role of consensus and the dangers of difficulties:
+	\begin{itemize}
+		\item Danger of permanent network splits.
+		\item Uncertainty in case of temporary forks.
+	\end{itemize}
+	
+\vspace{0.5 em}
+$\Rightarrow$ Both cases impact value of network to the users.
+
+\vspace{1.5 em}
+\uncover<2->{
+\textbf{Focus of this lecture:}
+	\begin{itemize}
+		\item General incentives for consensus relevant nodes (CRNs)
+		\item Bitcoin specifics and incentives driving consensus.
+		\item Consensus attacks in this context.
+	\end{itemize}
+}
+
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Economic Considerations of a CRN}
+
+To get participants to serve as compliant CRN and bear the corresponding cost, a Blockchain network typically offers revenues.
+
+\vspace{1 em}
+
+\footnotesize
+\textbf{General CRN P\&L categories}
+\begin{table}
+	{\renewcommand{\arraystretch}{1.3}%
+  \center
+  \begin{tabular}[]{p{0.43\textwidth} | p{0.47\textwidth}}
+		\textbf{Cost} & \textbf{Revenues}      \\
+		\hline
+		Computation & Block based (e.g., Coinbase tx)\\
+		\textit{(electricity, hardware, etc.)} & Transaction based (e.g.,fees)\\
+		Cost of stake* & Attestation rewards*\\
+		Cost of maintaining Authority* & Other Miner Extracted Value (MEV)
+  \end{tabular}}
+\end{table}
+
+\scriptsize *not applicable in Bitcoin context
+
+\vspace{1 em}
+\normalsize
+$\Rightarrow$ With revenues in network currency and cost often in FIAT, CRNs have an inherent incentive to keep network value - and consequently demand - high.
+	
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Computing Power Allocation}
+
+\begin{columns}[T]
+	\begin{column}{0.45\textwidth}
+		\textbf{Mining Market:} \\
+		\begin{itemize}
+			\item Competitive due to low entry barriers.
+			\item Profits only through above average efficiency.
+		\end{itemize}		
+		Under homogeneity, computing power will join or leave the network, until equilibrium is reached.
```

Comment:
```suggestion
		
```

Reply:
Could possibly be left out because it is already explained after that and on the figure.